packages/driver/cypress/e2e/e2e/origin/commands/files.cy.ts

```diff
@@ -64,11 +64,11 @@ context('cy.origin files', { browser: '!webkit' }, () => {
       })
 
       cy.shouldWithTimeout(() => {
-        const { consoleProps } = findCrossOriginLogs('readFile', logs, 'foobar.com')
+        const log = findCrossOriginLogs('readFile', logs, 'foobar.com')
```

Comment:
No functional change here. While working on these changes, I found it easier to read test failures when they were of the form "unable to read `consoleProps` from `null`" rather than "unable to destructure intermediate object" (the actual errors are significantly more verbose and harder to read than the above).